main.py

```diff
@@ -1,7 +1,44 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    Genre.create(
+        [
+            {"name": "Western"},
+            {"name": "Action"},
+            {"name": "Dramma"},
+        ]
+    )
+
+    Actor.create(
+        [
+            {"first_name": "George", "last_name": "Klooney"},
+            {"first_name": "Kianu", "last_name": "Reaves"},
+            {"first_name": "Scarlett", "last_name": "Keegan"},
+            {"first_name": "Will", "last_name": "Smith"},
+            {"first_name": "Jaden", "last_name": "Smith"},
+            {"first_name": "Scarlett", "last_name": "Johansson"}
+        ]
+    )
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(
+        first_name="George",
+        last_name="Klooney"
+    ).update(last_name="Clooney")
+    Actor.objects.filter(
+        first_name="Kianu",
+        last_name="Reaves"
+    ).update(first_name="Keanu", last_name="Reeves")
+
+    Genre.objects.filter(name="Action").delete()
+    Actor.objects.filter(first_name="Scarlett").delete()
+
+    return Actor.objects.filter(last_name="Smith").order_by("first_name")
```

Comment:
By general practice, all operations with model objects should be performed via a special `Manager` class that is accessible via the class attribute `objects`. In case you want to add the functionality that is related to a model object operations, you should create the new `Manager` class and extend it there. However, it is possible to use Mixin here because you add the same functionality to multiple classes, it is just a note so that you to remember what should be a first choice in such cases.